lxd/instance/instance_utils.go

```diff
@@ -128,6 +128,22 @@ func ValidConfig(sysOS *sys.OS, config map[string]string, expanded bool, instanc
 }
 
 func validConfigKey(os *sys.OS, key string, value string, instanceType instancetype.Type) error {
+	// Check if the key is a valid prefix and whether or not it requires a subkey.
+	knownPrefixes := append(instancetype.ConfigKeyPrefixesAny, instancetype.ConfigKeyPrefixesContainer...)
+	if strings.HasSuffix(key, ".") {
```

Comment:
in what scenario would this occur?

Reply:
If a key such as `linux.sysctl.` is passed in, it is invalid since there is no subkey provided. 

For this check, we check if a known key is passed in, if so, we inform the user that the key requires a subkey. Otherwise, the key is unknown.